migration-from-4x.md

```diff
@@ -308,7 +312,9 @@ editor.addButton('mybutton', {
 ```js
 editor.ui.registry.addButton('myButton', {
   text: 'My Button',
-  onAction: (buttonApi) => alert('My Button clicked!')
+  onAction: function (buttonApi) {
+    alert('My Button clicked!')
```

Comment:
Missing semicolon at the end of the line here.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d